lib/config/create-urc.js

```diff
@@ -38,6 +38,30 @@ class Urc {
       deleteCache: true
     });
   }
+
+  getClientEnvVars() {
+    // Inspired from https://github.com/facebook/create-react-app/blob/next/packages/react-scripts/config/env.js#L71
+    // Grabs NODE_ENV and environment variables that startWith CLIENT_ENV_PREFIX and prepare them to be
+    // injected into the application via EnvironmentPlugin in Webpack configuration.
```

Comment:
I see I reviewed a couple of PRs in the wrong order —— see my comments on the other PR for my reservations about requiring a prefix for these env variables.